### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by rmuolic**
Initial edit of doc code owners for the OCS repo.
Please edit as needed @digitaljoshua , @afloresOSI , @VictoriaTouati-AVEVA , @osisoft-mbishop .

### CODE REVIEW

**Comment by afloresOSI** [comment]
What about lines 10-14 and line 16?

**Comment by rmuolic** [comment]
> What about lines 10-14 and line 16?

I don’t know who to assign for these lines, thus my request to edit as needed. 👍🏼 @afloresOSI

File: CODEOWNERS [hunk 1]
```diff
@@ -6,18 +6,19 @@
 content/api-reference/sequential-data-store  @digitaljoshua
 content/api-reference/identity @digitaljoshua
 content/api-reference/management @digitaljoshua
-content/api-reference/data-views @osimarkrogers
+content/api-reference/data-views @rmuolic
 content/api-reference/metadata @osimarkrogers
 content/api-reference/operations @osimarkrogers
 content/api-reference/data-ingress @osimarkrogers
 content/api-reference/assets @lassato-OSI
```
Comment (afloresOSI) [inline CODEOWNERS:13]:
```suggestion
content/api-reference/assets @afloresOSI 
```

File: CODEOWNERS [hunk 2]
```diff
@@ -6,18 +6,19 @@
...
 content/api-reference/asset-rules @lassato-OSI
```
Comment (afloresOSI) [inline CODEOWNERS:14]:
```suggestion
content/api-reference/asset-rules @afloresOSI 
```

File: CODEOWNERS [hunk 3]
```diff
@@ -6,18 +6,19 @@
...
 content/add-organize-data @VictoriaTouati-AVEVA @afloresOSI
 content/communities @dwicinas-osisoft
```
Comment (afloresOSI) [inline CODEOWNERS:16]:
```suggestion
content/communities @osisoft-mbishop
```

**Comment by afloresOSI** [comment]
> > What about lines 10-14 and line 16?
> 
> I don’t know who to assign for these lines, thus my request to edit as needed. 👍🏼 @afloresOSI

My fault, I didn't read your note. I updated the items I knew. We should probably pick a default person for the items we aren't sure about, just to make sure we remove those who have left.

**Pull request merged**